packages/DateTimeInput/src/index.tsx

```diff
@@ -95,8 +95,7 @@ const DateTimeInput: FunctionComponent<IDateTimeInputProps> = ({
         id={id}
         onChange={handleChange}
         onFocus={() => setOpen(true)}
-        showTimeInput={whitelistedType === TYPE_DATE_TIME}
-        showTimeSelect={whitelistedType === TYPE_TIME}
+        showTimeInput={[TYPE_TIME, TYPE_DATE_TIME].includes(whitelistedType)}
         showTimeSelectOnly={whitelistedType === TYPE_TIME}
```

Comment:
We should remove this line, otherwise you'll get the broken time selector appearing. The option `showTimeSelectOnly` only works together with `showTimeSelect`

Reply:
`showTimeSelectOnly` is needed to hide the calendar without it you get this
<img width="278" alt="screenshot" src="https://user-images.githubusercontent.com/35026928/101759194-32cfff00-3ad1-11eb-8822-fa6c2d814fb1.png">

With it you get
<img width="218" alt="screenshot" src="https://user-images.githubusercontent.com/35026928/101759232-3bc0d080-3ad1-11eb-994d-51a0861275d6.png">

The input styling may need some tweaks but we can check that in different usages.

Reply:
Ah, cool, thanks. Without the option `showTimeInput` it looked broken, I hadn't tested it with.